reservations/models.py

```diff
@@ -10,3 +10,6 @@ class Reservation(models.Model):
     end_time = models.DateTimeField()
     other_occupants = models.TextField()
     purpose = models.TextField()
+
+    def __str__(self):
+        return self.room.name
```

Comment:
This is already what the room will look like as a string - so they'll effectively have the exact same name. I would suggest something like: 

```python
f"{self.user}'s reservation in {self.room}"
```

or somethin' like that

Reply:
I fixed it to what you recommended. I remember reading about the f"___" thing, but I don't remember the specifics. Can you point me toward the documentation or give me a google keyword?